packages/core/botpress/src/web/components/Util/BackendToast.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import { toast } from 'react-toastify'
+import EventBus from '~/util/EventBus'
+
+export default class BackendToast extends Component {
+  constructor(props) {
+    super(props)
+
+    const uniqueCodes = [...new Set(props.codes)]
+    uniqueCodes.forEach(code => {
+      let id = null
+
+      EventBus.default.on(`toast.${code}`, options => {
+        options.text = options.text.length > 64 ? `${options.text.slice(0, 63)}...` : options.text
```

Comment:
@dmytropanontko , I think checking `options.text.length` is redundant. You can always slice required length?

Reply:
@epaminond I'm checking length to know need I add `...` or no

Reply:
Oh, OK.